src/v/serde/test/serde_test.cc

```diff
@@ -703,6 +726,8 @@ struct new_no_cs
 
     serde::checksum_t unchecked_dummy_checksum_{0U};
     std::vector<test_msg1_new_manual> data_;
+
+    auto serde_fields() { return std::tie(unchecked_dummy_checksum_, data_); }
```

Comment:
this one has them flipped the other way, so maybe its intentional

Reply:
oh i see. yeah, those value types have something to do with operator overloading, but the need (or lack of need, as the case may be) for those operators is mysterious to me.